addon/components/nypr-card/button.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+import layout from '../../templates/components/nypr-card/button';
+
+const NyprCardButton = Ember.Component.extend({
+  layout,
+  tagName: 'button',
+  classNames: ['nypr-card-button'],
+  'data-test-selector': 'nypr-card-button'
+});
+
+NyprCardButton.reopen({
+  attributeBindings: ['data-test-selector']
+});
```

Comment:
I don't think we need to reopen the component to set attributeBindings when we have access to the component definition. This property should work in the `Ember.Component.extend()` block above.